RadarSDK/RadarAPIHelper.m

```diff
@@ -78,7 +78,12 @@ - (void)requestWithMethod:(NSString *)method
                 configuration.timeoutIntervalForResource = 10;
             }
 
+            NSDate *requestStart = [NSDate date];
+
             void (^dataTaskCompletionHandler)(NSData *data, NSURLResponse *response, NSError *error) = ^(NSData *data, NSURLResponse *response, NSError *error) {
+                NSTimeInterval latency = [requestStart timeIntervalSinceNow] * -1;
```

Comment:
didn't the multiplication for a bit, maybe add a comment that timeIntervalSinceNow returns a negative number?